tests/linearGaussian_snpe_test.py

```diff
@@ -188,10 +197,12 @@ def simulator(theta):
     samples = posterior.sample((num_samples,))
 
     # Compute the c2st and assert it is near chance level of 0.5.
-    check_c2st(samples, target_samples, alg=f"npe_{density_estimator}")
+    check_c2st(samples, target_samples, alg=f"{npe_method_str}_{density_estimator}")
 
 
-def test_c2st_npe_on_linearGaussian_different_dims(density_estimator="maf"):
+def test_c2st_npe_on_linearGaussian_different_dims(
```

Comment:
I think this as well can be reverted, and should not test for `npe_b` explicitly